### ISSUE TITLE
Move into d2 repo

### ISSUE THREAD

**Issue by nhooyr** [terrastruct/d2-docs#17]

**Comment by alixander** [comment]
i don't think we should. i wanna put up a landing page and other pages in here, so it's just the site for d2

**Comment by nhooyr** [comment]
It'll be easier to manage CI/versioning/updating/everything when it's all in one repo.

e.g. PRs that update the syntax can update the docs at the same time.

I don't see how adding a landing page here would make it less suitable for integration into the core repo.
